Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -399,6 +402,17 @@ bool PPCSymbolDB::LoadMap(const Core::CPUThreadGuard& guard, const std::string&
     if (name[strlen(name) - 1] == '\r')
       name[strlen(name) - 1] = 0;
 
+    // Split the current name string into separate parts, and get the object name
+    // if it exists.
+    std::vector<std::string> parts = SplitString(name, '\t');
+    size_t num_parts = parts.size();
+
+    const std::string name_string = std::string(StripWhitespace(parts[0]));
+    const std::string object_filename_string =
+        num_parts > 1 && !parts[1].starts_with("Linker Generated Symbol File") ?
+            std::string(StripWhitespace(parts[1])) :
+            "";
```

Comment:
I do believe it's a valuable information. so I'm not a huge fan of dropping `Linker Generated Symbol File` (plus it will truncate existing symbol maps). This follows along my previous comment regarding the filtering using a dot character as it truncated objects when there were more than one (e.g. a library file). This filtering behavior truncates the resulting Dolphin map when saved back which isn't good.